ui_testing/testcases/header_tests/test08_usermanagement.py

```diff
@@ -238,4 +227,289 @@ def test010_update_user_role_with_save_cancel_btn(self, save):
                 ' + Assert {} (current_role) == {} (user_role)'.format(current_role, user_role))
             self.assertEqual(current_role, user_role)
 
+    @parameterized.expand(['save_btn', 'cancel'])
+    def test010_update_user_email_with_save_cancel_btn(self, save):
+        """
+        User management: I can update user email with save & cancel button
+        LIMS-6397
+        :return:
+        """
+        self.base_selenium.click(element='header:user_management_button')
+        # open random user in the edit mode
+        self.header_page.get_random_user()
+        user_url = self.base_selenium.get_url()
+        self.base_selenium.LOGGER.info(' + user_url : {}'.format(user_url))
+        self.order_page.sleep_tiny()
+        current_email = self.header_page.get_user_email()
```

Comment:
use API list users

Reply:
@0xIslamTaha why I use this , I will change the create user from API then filter by it what is the problem in that

Reply:
@dianaeldemerdash This test case could edit the `admin` use and it shouldnt